deploy/helm/sumologic/values.yaml

```diff
@@ -353,6 +353,27 @@ sumologic:
       otelcol:
         ## Extra processors for container logs. See https://help.sumologic.com/docs/send-data/kubernetes/collecting-logs/ for details.
         extraProcessors: []
+        ## useDefaultExporters uses default helm chart exporters.
+        ## depending on the configuration it may be:
+        ## * sumologic/containers and/or sumologic/sumologic-mock-containers
+        ## * sumologic and/or sumologic/sumologic-mock
+        useDefaultExporters: true
+        ## routing defines how to use extraExporters
+        routing:
+          []
+          ## -
+          ##  exporter is an exporter name. For example `exporter/name` for the following extraExporters configuration:
+          ##
+          ##  extraExporters:
+          ##    exporter/name:
+          ##      config_key: config_value
+          ##  exporter: sumologic
+          ##  ## set to true if the exporter should be used as default exporter
+          ##  default: true
```

Comment:
Instead of this, I'd prefer to have a `otelcol.defaultExporter` setting, and make it mutually exclusive with `useDefaultExporters`.

Reply:
Could you provide a configuration example? I'm not sure what would `defaultExporter` have

Reply:
```yaml
otelcol:
  defaultExporter: sumologic/new
  routing:
     - exporter: sumologic/test
       statement: route() where resource.attributes["exporter"] == "test"
extraExporters:
  sumologic/new:
    endpoint: https://new-endpoint
  sumologic/test:
    endpoint: https://test-endpoint
```

Reply:
Yes, it is possible, but it will look like the following:
```
otelcol:
  defaultExporters:
    - sumologic/new
    - debug/new
  routing:
     - exporter: sumologic/test
       statement: route() where resource.attributes["exporter"] == "test"
extraExporters:
  sumologic/new:
    endpoint: https://new-endpoint
  debug/new:
    endpoint: https://new-endpoint
  sumologic/test:
    endpoint: https://test-endpoint
```

And that way users needs to maintain 3 configuration keys

Also it requires users to know what our default exporters are, if they would like to send logs to S3 for example

Reply:
Also this is more complicated for logs, as we may have different exporter per type (systemd, kubelet, container) for http source, so `useDefaultExporters` is easier for us to maintain. We may migrate off it in v5, but for v4 I wouldn't expose default exporters as they depends on other configuration